include/signal.h

```diff
@@ -37,20 +37,28 @@
 
 /* Signal set management definitions and macros. */
 
-#define NULL_SIGNAL_SET ((sigset_t)0x00000000)
-#define ALL_SIGNAL_SET  ((sigset_t)0xffffffff)
 #define MIN_SIGNO       1               /* Lowest valid signal number */
-#define MAX_SIGNO       31              /* Highest valid signal number */
+#define MAX_SIGNO       63              /* Highest valid signal number */
 #define GOOD_SIGNO(s)   ((((unsigned)(s)) <= MAX_SIGNO))
```

Comment:
Maybe reasonable to
```suggestion
#define GOOD_SIGNO(s)   ((((unsigned)(s)) >= MIN_SIGNO) && (((unsigned)(s)) <= MAX_SIGNO))
```
? As `0` is not a valid signal

Reply:
I don't think that you can do that because zero is a valid signal number to.  It is the do nothing signal.  This change would be good in most places, but we cannot do this without also adding a special case for logic in all of the places where a zero signal is acceptable.

This GOOD_SIGNO macro has always accepted the zero signal as good for this reason.  I don't want to do that right now, because I don't want to introduce any other unrelated bad behavior.

I will resolve this if you agree.

Reply:
Yes. I will look into the code again. I was thinking that zero value is filtered out at signal API level and does not reach the place where this macro is used. But it seems I'm wrong